docs/source/conf.py

```diff
@@ -0,0 +1,231 @@
+# -*- coding: utf-8 -*-
+#
+# Configuration file for the Sphinx documentation builder.
+#
+# This file does only contain a selection of the most common options. For a
+# full list see the documentation:
+# http://www.sphinx-doc.org/en/stable/config
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+# import recommonmark
+# from recommonmark.parser import CommonMarkParser
+# from recommonmark.transform import AutoStructify
+
+# -- Project information -----------------------------------------------------
+
+project = 'GovReady-Q Compliance Server'
+copyright = '2018, GovReady PBC'
+author = 'Josh Tauberer, Greg Elin'
+
+github_doc_root = 'https://github.com/GovReady/govready-q/tree/master/docs'
+
+# The short X.Y version
+version = '0.8.2'
```

Comment:
Version information should be tied to how the rest of our build process works. Although it wouldn't make sense now because our VERSION file is auto-generated during the build and doesn't exist otherwise. So we need to improve that.

Reply:
We need to take this tech debt and a future issue.